diff --git a/code/game/g_local.h b/code/game/g_local.h
index b9931c5..6495a5c 100644
--- a/code/game/g_local.h
+++ b/code/game/g_local.h
@@ -30,7 +30,11 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 //==================================================================
 
 // the "gameversion" client command will print this plus compile date
+#ifdef NEONARENA_MOD
+#define	GAMEVERSION	"NeonArena-0.1"
+#else
 #define	GAMEVERSION	BASEGAME
+#endif
 
 #define BODY_QUEUE_SIZE		8
 
