diff --git a/Makefile b/Makefile
--- a/Makefile
+++ b/Makefile
@@ -2405,6 +2405,7 @@ Q3GOBJ_ = \
   $(B)/$(BASEGAME)/game/g_items.o \
   $(B)/$(BASEGAME)/game/g_killspree.o \
   $(B)/$(BASEGAME)/game/g_misc.o \
+  $(B)/$(BASEGAME)/game/g_neonwave.o \
   $(B)/$(BASEGAME)/game/g_missile.o \
   $(B)/$(BASEGAME)/game/g_mover.o \
   $(B)/$(BASEGAME)/game/g_playerstore.o \
@@ -2466,6 +2467,7 @@ MPGOBJ_ = \
   $(B)/$(MISSIONPACK)/game/g_items.o \
   $(B)/$(MISSIONPACK)/game/g_killspree.o \
   $(B)/$(MISSIONPACK)/game/g_misc.o \
+  $(B)/$(MISSIONPACK)/game/g_neonwave.o \
   $(B)/$(MISSIONPACK)/game/g_missile.o \
   $(B)/$(MISSIONPACK)/game/g_mover.o \
   $(B)/$(MISSIONPACK)/game/g_playerstore.o \
